app/shared/connectors/BaseDownstreamConnector.scala

```diff
@@ -87,7 +90,8 @@ trait BaseDownstreamConnector extends Logging {
     val strategy = uri.strategy
 
     def doDelete(implicit hc: HeaderCarrier): Future[DownstreamOutcome[Resp]] = {
-      http.DELETE(getBackendUri(uri.path, strategy))
+      val fullUrl = UrlUtils.appendQueryParams(getBackendUri(uri.path, strategy), queryParams)
```

Comment:
the doDelete method can be updated to: `http.delete(url"${getBackendUri(uri.path, strategy)}").execute`